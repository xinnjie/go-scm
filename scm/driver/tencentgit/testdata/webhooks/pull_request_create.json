{
  "object_kind": "merge_request",
  "user": {
    "name": "git_user2",
    "username": "git_user2",
    "avatar_url": "https://blog.bobo.com.cn/s/blog_6e572cd60101qls0.html"
  },
  "object_attributes": {
    "id": 23,
    "target_branch": "master",
    "source_branch": "fenzhi01",
    "source_project_id": 2,
    "author_id": 3,
    "assignee_id": 4,
    "title": "agree",
    "created_at": "2018-03-13T09:51:06+0000",
    "updated_at": "2018-03-13T09:51:06+0000",
    "st_commits": null,
    "st_diffs": null,
    "milestone_id": null,
    "state": "opened",
    "merge_status": "unchecked",
    "target_project_id": 14,
    "iid": 1,
    "description": "",
    "source": {
      "name": "rename",
      "ssh_url": "ssh://git@tencent.com/z-413/rename.git",
      "http_url": "http://tencent.com/z-413/rename.git",
      "web_url": "https://tencent.com/z-413/rename",
      "visibility_level": 20,
      "namespace": "z-413"
    },
    "target": {
      "name": "rename",
      "ssh_url": "ssh://git@tencent.com/z-413/rename.git",
      "http_url": "http://tencent.com/z-413/rename.git",
      "web_url": "https://tencent.com/z-413/rename",
      "visibility_level": 20,
      "namespace": "z-413"
    },
    "last_commit": {
      "id": "1480a4610ca01dd10cb5bc30d83151eab98c09a1",
      "message": "fixed readme",
      "timestamp": "2012-01-03T23:36:29+02:00",
      "url": "https://tencent.com/z-413/rename/commits/da1480a4610ca01dd10cb5bc30d83151eab98c09a1",
      "author": {
        "name": "alex",
        "email": "alex@tencent.com"
      }
    },
    "url": "https://tencent.com/rename/merge_requests/1",
    "action": "open",
    "extension_action": "open"
  }
}