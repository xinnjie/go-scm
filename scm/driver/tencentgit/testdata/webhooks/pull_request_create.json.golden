{
  "Action": "opened",
  "Repo": {
    "ID": "14",
    "Namespace": "z-413",
    "Name": "rename",
    "FullName": "z-413/rename",
    "Perm": null,
    "Branch": "",
    "Private": false,
    "Clone": "http://tencent.com/z-413/rename.git",
    "CloneSSH": "ssh://git@tencent.com/z-413/rename.git",
    "Link": "https://tencent.com/z-413/rename",
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
  },
  "PullRequest": {
    "Number": 1,
    "Title": "agree",
    "Body": "description",
    "State": "open",
    "Sha": "1480a4610ca01dd10cb5bc30d83151eab98c09a1",
    "Ref": "refs/merge-requests/1/head",
    "Base": {
      "Ref": "master",
      "Repo": {
        "ID": "14",
        "Namespace": "z-413",
        "Name": "rename",
        "FullName": "z-413/rename",
        "Perm": null,
        "Branch": "",
        "Private": false,
        "Clone": "http://tencent.com/z-413/rename.git",
        "CloneSSH": "ssh://git@tencent.com/z-413/rename.git",
        "Link": "https://tencent.com/z-413/rename",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
      }
    },
    "Head": {
      "Ref": "fenzhi01",
      "Sha": "1480a4610ca01dd10cb5bc30d83151eab98c09a1",
      "Repo": {
        "ID": "2",
        "Namespace": "z-413",
        "Name": "rename",
        "FullName": "z-413/rename",
        "Perm": null,
        "Branch": "",
        "Private": false,
        "Clone": "http://tencent.com/z-413/rename.git",
        "CloneSSH": "ssh://git@tencent.com/z-413/rename.git",
        "Link": "https://tencent.com/z-413/rename",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
      }
    },
    "Source": "fenzhi01",
    "Target": "master",
    "Fork": "z-413/rename",
    "Link": "https://tencent.com/rename/merge_requests/1",
    "Closed": false,
    "Merged": false,
    "Author": {
      "Login": "git_user2",
      "Name": "git_user2",
      "Email": "",
      "Avatar": "https://blog.bobo.com.cn/s/blog_6e572cd60101qls0.html"
    },
    "Created": "2018-03-13T09:51:06Z",
    "Updated": "2018-03-13T09:51:06Z"
  },
  "Sender": {
    "Login": "git_user2",
    "Name": "git_user2",
    "Email": "",
    "Avatar": "https://blog.bobo.com.cn/s/blog_6e572cd60101qls0.html"
  }
}
